### ISSUE TITLE
Hardware field needs to be updated

### ISSUE THREAD

**Issue by wilzbach** [CyberShadow/bugzilla-meta#1]
![image](https://user-images.githubusercontent.com/4370550/37940530-7a7ee83a-3169-11e8-9906-73022325b4a2.png)

Probably should be x86, x86_64 and other?

**Comment by CyberShadow** [comment]
Yep, but this is a configuration issue. Will be dealt with at the same time when we create products/components.
